.github/workflows/dev-cicd.yml

```
@@ -1,4 +1,4 @@
-name: CI/CD Dev
+계획name: CI/CD Dev
```

actionlint (1.7.11)

[error] unexpected key "계획name" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)